infrastructure/zk/src/init.ts

```diff
@@ -156,12 +153,12 @@ export const initDevCmdAction = async ({
     }
     let deploymentMode = validiumMode !== undefined ? contract.DeploymentMode.Validium : contract.DeploymentMode.Rollup;
     await initSetup({ skipEnvSetup, skipSubmodulesCheckout, runObservability, deploymentMode });
-    await initDatabase({ skipVerifierDeployment: false });
+    await deployVerifier();
```

Comment:
Why did we get rid of the `skipVerifierDeployment` flag?

Reply:
just doing some refactoring.

I am not sure why we include the verifier smart contract deployment in the method `initDatabase` ?

plus because of this flag (?) we are running the `init database` twices in the script which take times when doing it on a remote db

Reply:
Makes sense, but there's no need to deploy contracts on L1 if we run reinit for example

I think it's better to have a conditional check for `deployVerlifier`, however outside of initDatabase
Or I might be missing some context

Reply:
sure, adding a flag to skip when reiniting